conftest.py

```diff
@@ -0,0 +1,26 @@
+def pytest_addoption(parser):
+    # The following flags are used by benchmark tests
+    parser.addoption(
+        "--num_exercises",
+        action="store",
+        default="1",
+        help="Number of exercises to run",
+    )
+    parser.addoption(
+        "--max_iterations",
+        action="store",
+        default="1",
+        help="Number of times to rerun mentat with error messages",
+    )
+    parser.addoption(
+        "--max_workers",
+        action="store",
+        default="1",
+        help="Number of workers to use for multiprocessing",
+    )
+    parser.addoption(
+        "--start_at",
+        action="store",
+        default="0",
+        help="Number of workers to use for multiprocessing",
```

Comment:
which exercise to start at? maybe a more general way to say since this would be used for other benchmark sets as well

Reply:
I added this flag because a few times the benchmark would fail half way through with a spurious error and I needed an easy way to restart. Since then I added a try catch block around run so I haven't run into that problem as much and I think maybe I'll replace this flag with a `--continue` flag which when set the test won't git reset and directories that already have test_output.txt will be skipped.